test/mode/modeNormal.test.ts

```diff
@@ -268,6 +268,14 @@ suite("Mode Normal", () => {
     });
 
     newTest({
+      title: "Can handle 'ci[' spanning multiple lines",
```

Comment:
Yes, I wasn't sure about the level of testing desired, given that the underlying functionality (character matching) is already covered. Happy to add more tests (and that's generally my default), but wanted to be sensitive to test run times if that's a concern for this project.